src/Result.mo

```diff
@@ -14,7 +14,7 @@ module {
 ///
 /// For example, given a function `createUser(user : User) : Result<Id, String>`
 /// where `String` is an error message we could use it like so:
-/// ```motoko
```

Comment:
This is what I was talking about in that other PR. We're now losing syntax highlighting, because we have a snippet of code that isn't runnable on its own. Do you see a way around this? Would we need to teach `mo-doc` to emit the right asciidoc source code blocks?

Reply:
Interesting, nevermind this comment then.